app/static/css/header.css

```
@@ -106,3 +106,95 @@ header {
     width: 100%;
     flex: 1
 }
+
+.desktop-tap-div {
```

Typo here (mentioned below as well)

```suggestion
.desktop-tab-div {
```